test/addon_spec.rb

```diff
@@ -146,6 +146,20 @@
     end
   end
 
+  describe "#beta_release?" do
+    it "returns if addon is beta release" do
```

Comment:
NP: I'd split it to two tests like `it returns true if addon is beta release` and `it returns false if addon is not beta release`.